dl4j-labs/src/main/java/ai/certifai/training/classification/HeartDiseaseBinaryClassification.java

```diff
@@ -0,0 +1,229 @@
+ * The code for dataset loading and preparation was done for you.
+ * Also, the code to iterate through training epochs and evaluating the model were provided.
+ *
+ * Please complete this exercise by configuring a neural network that can perform a binary classification task.
+ * Take special note of the different input and output number that is required in order to run the dataset
+ * successfully.
+ *
+ * Remember to uncomment certain parts of code to run the entire script.
+ *
+ * *****************************************************
+ */
+
+public class HeartDiseaseBinaryClassification {
+    private static final int TOTAL_DATA = 303;
+    private static final double RATIO_TRAIN_TEST_SPLIT = 0.8;
+
+    // Training info
+    private static final int EPOCH = 1000;
+
+    public static void main(String[] args) throws Exception {
+
+        //=====================================================================
+        //            Step 1: Load & Transform data
+        //=====================================================================
+
+        RecordReader rr = loadData();
+
+        List<List<Writable>> rawTrainData = new ArrayList<>();
+        List<List<Writable>> rawTestData = new ArrayList<>();
+
+        // Get total length of data
+        int numTrainData = (int) Math.round(RATIO_TRAIN_TEST_SPLIT * TOTAL_DATA);
+        int idx = 0;
+        while (rr.hasNext()) {
+            if(idx < numTrainData) {
+                rawTrainData.add(rr.next());
+            } else {
+                rawTestData.add(rr.next());
+            }
+            idx++;
+        }
+
+        System.out.println("Total train Data " + rawTrainData.size());
+        System.out.println("Total test Data " + rawTestData.size());
+
+        List<List<Writable>> transformedTrainData = transformData(rawTrainData);
+        List<List<Writable>> transformedTestData = transformData(rawTestData);
+
+        DataSetIterator trainData = makeIterator(transformedTrainData);
+        DataSetIterator testData = makeIterator(transformedTestData);
+
+        DataNormalization normalizer = new NormalizerStandardize();
+        normalizer.fit(trainData);
+        trainData.setPreProcessor(normalizer);
+        testData.setPreProcessor(normalizer);
+
+        //=====================================================================
+        //            Step 2: Define Model
+        //=====================================================================
+
+//        There are subtle differences in model configuration for different classification task.
+//        Since this is a binary classification task, please remember to configure the correct number of outputs for
+//        output layer. With this said, please configure a model using your own learning rate, activation function,
+//        and number of neural network layer and nodes. Feel free to experiment with it to achieve best result.
+
```

Comment:
Suggestion: 
It may a good idea to provide:
1. A clear indication/instruction  for student to identify which part of the code at the specific line requires their input or make changes in the code (uncomment the line with the provided code)
e.g. // #### Please insert your code here / START ####

//  #### END ####

2. The first line for student to kickstart in model configuration. i.e. : 
MultiLayerConfiguration conf = new NeuralNetConfiguration.Builder()
                .seed(1234)
// #### Configure your model here ####

Reply:
Done as per suggested @ameliepeter

Reply:
Also done as per suggested @ameliepeter to imporve model perormance

Reply:
@ameliepeter I have done the necessaries. Thank you.

Reply:
Thanks a lot !! @KianYang-Lee